core/src/layers/prometheus_client.rs

```diff
@@ -160,47 +156,63 @@ impl EncodeLabelSet for ErrorLabels {
 /// [`PrometheusClientMetricDefinitions`] provide the definition about RED(Rate/Error/Duration) metrics with the `prometheus-client` crate.
 #[derive(Debug, Clone)]
 struct PrometheusClientMetricDefinitions {
-    /// Total counter of the specific operation be called.
+    /// Total times of the specific operation be called.
     requests_total: Family<OperationLabels, Counter>,
-    /// Total counter of the errors.
+    /// Total times of the specific operation be called but meet error.
     errors_total: Family<ErrorLabels, Counter>,
     /// Latency of the specific operation be called.
     request_duration_seconds: Family<OperationLabels, Histogram>,
-    /// The histogram of bytes
-    bytes_histogram: Family<OperationLabels, Histogram>,
-    /// The counter of bytes
+
+    /// Total size of bytes.
     bytes_total: Family<OperationLabels, Counter>,
+    /// The size of bytes.
+    bytes: Family<OperationLabels, Histogram>,
 }
 
 impl PrometheusClientMetricDefinitions {
     pub fn register(registry: &mut Registry) -> Self {
         let requests_total = Family::default();
         let errors_total = Family::default();
-        let bytes_total = Family::default();
         let request_duration_seconds = Family::<OperationLabels, _>::new_with_constructor(|| {
             let buckets = histogram::exponential_buckets(0.01, 2.0, 16);
             Histogram::new(buckets)
         });
-        let bytes_histogram = Family::<OperationLabels, _>::new_with_constructor(|| {
+
+        let bytes_total = Family::default();
+        let bytes = Family::<OperationLabels, _>::new_with_constructor(|| {
             let buckets = histogram::exponential_buckets(1.0, 2.0, 16);
             Histogram::new(buckets)
         });
 
-        registry.register("opendal_requests", "", requests_total.clone());
-        registry.register("opendal_errors", "", errors_total.clone());
+        registry.register(
+            "opendal_requests",
```

Comment:
@Xuanwo What do you think of removing the `opendal_` prefix?
Users can customize prefix through the following code:

```rust
let builder = services::Memory::default();
let mut registry = prometheus_client::registry::Registry::default();

let op = Operator::new(builder)
        .expect("must init")
        .layer(PrometheusClientLayer::new(registry.sub_registry_with_prefix("opendal")))
        .finish();
```

Reply:
But for `PrometheusLayer` (not `PrometheusClientLayer`), there is no builtin `opendal_` prefix in the metrics.

Reply:
Actually, I want the metrics exported by these two layers to be the same, the only difference being the different libraries they use, but that is not the case now.

Reply:
I prefer to add prefix for `PrometheusLayer`

Reply:
Ok, I will consider unifying the metrics of these two prometheus layers in a later PR.